pkg/collector/corechecks/cluster/kubernetes_eventbundle.go

```diff
@@ -132,6 +132,19 @@ func (b *kubernetesEventBundle) formatEvents(clusterName string, providerIDCache
 	return output, nil
 }
 
+func addKindRelatedTag(kind, name string) string {
```

Comment:
could we add `Statefulset` ? I'm comparing to what we do in the tagger https://github.com/DataDog/datadog-agent/blob/master/pkg/tagger/collectors/kubelet_extract.go#L112-L122

Reply:
Also thinking, what about adding a generic `kube_lower($kind)` tag? Would work quite well (we might still need to add `pod_name` as it's non standard

Reply:
+1

Reply:
ok, the card was just for podname initially, but i will add that

Reply:
feel free to backlog it since it's out of scope